src/App/App.js

```diff
@@ -9,11 +8,28 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      allMovies: movieData,
-      specificMovie: null
+      specificMovie: null,
+      isLoading: true,
+      error: null
     };
   }
   
+  fetchCall = () => {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+      .then(response => response.json())
+      .then(data => 
+        this.setState({
+          allMovies: data,
+          specificMovie: null,
+          isLoading: false
```

Comment:
I loved this loading feature. Very impressive! Now that we're fetching, we can delete the movie data file!